packages/grafana-data/src/themes/colorManipulator.ts

```diff
@@ -66,6 +68,16 @@ export function rgbToHex(color: string) {
   return `#${values.map((n: number) => intToHex(n)).join('')}`;
 }
 
+/**
+ * Converts a color to hex6 format if there is no alpha, hex8 if there is.
+ * @param color - Hex, RGB, HSL color
+ * @returns A hex color string, i.e. #ff0000 or #ff0000ff
+ */
+export function formatHexByAlpha(color: string): string {
```

Comment:
```suggestion
export function asHexString(color: string): string {
```
This will eventually be called as `colorManipulator.asHexString(...)` so I don't think we need color in the name

I wonder if we should short circuit if the color starts with '#'?  ie: if color[0] === '#' return color?